src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -30,20 +30,20 @@ def setUp(self):
     def tearDown(self):
         pass
     
-    def assertEqualToIfExistDeviceOrPreloadedCondtionalParam(self, attr_name, record,\
-                 preloaded_condtionals, registration_request):
+    def assertEqualToIfExistDeviceOrPreloadedConditionalParam(self, attr_name, record,\
+                 preloaded_Conditionals, registration_request):
```

Comment:
Please  use `preloaded_conditionals` instead of `preloaded_Conditionals` (lowercase 'c') to match our conventions (here and below).